README.md

```diff
@@ -3,93 +3,100 @@
+### Fine-tuning recipes
 
-| Model                                         | Sizes     |   Finetuning Methods |
-|-----------------------------------------------|-----------|-----------------------------------------------------------|
-| [Llama2](torchtune/models/llama2/_model_builders.py)   | 7B        | Full Finetuning [[single device low memory](recipes/configs/llama2/7B_full_low_memory.yaml),  [distributed](recipes/configs/llama2/7B_full.yaml)] LoRA [[single device](recipes/configs/llama2/7B_lora_single_device.yaml),  [distributed](recipes/configs/llama2/7B_lora.yaml)] QLoRA [[single device](recipes/configs/llama2/7B_qlora_single_device.yaml)] |
-| [Llama2](torchtune/models/llama2/_model_builders.py)   | 13B       | [Full Finetuning](recipes/configs/llama2/13B_full.yaml), [LoRA](recipes/configs/llama2/13B_lora.yaml)
-| [Mistral](torchtune/models/mistral/_model_builders.py)   | 7B       | Full Finetuning [[single device low memory](recipes/configs/mistral/7B_full_low_memory.yaml), [distributed](recipes/configs/mistral/7B_full.yaml)] LoRA [[single device](recipes/configs/mistral/7B_lora_single_device.yaml), [distributed](recipes/configs/mistral/7B_lora.yaml)] QLoRA [[single device](recipes/configs/mistral/7B_qlora_single_device.yaml)]
-| [Gemma](torchtune/models/gemma/_model_builders.py)   | 2B        | [Full Finetuning](recipes/configs/gemma/2B_full.yaml)
+torchtune provides the following fine-tuning recipes.
 
+| Training                           | Fine-tuning Method                 |
+|------------------------------------|------------------------------------|
+| Distributed Training [1 to 8 GPUs] | Full [[code](recipes/full_finetune_distributed.py), [example](recipes/configs/llama2/13B_full.yaml)], LoRA [[code](recipes/lora_finetune_distributed.py), [example](recipes/configs/llama2/13B_lora.yaml)] |
+| Single Device / Low Memory [1 GPU] | Full [[code](recipes/full_finetune_distributed.py), [example](recipes/configs/llama2/13B_full.yaml)], LoRA + QLoRA [[code](recipes/lora_finetune_distributed.py), [example](recipes/configs/llama2/13B_lora.yaml)] |
+| Single Device [1 GPU]              | DPO [[code](recipes/full_finetune_distributed.py), [example](recipes/configs/llama2/13B_full.yaml)]
 
 &nbsp;
 
-### Finetuning resource requirements
 
-Note: These resource requirements are based on GPU peak memory reserved during training using the specified configs. You may
-experience different peak memory utilization based on changes made in configuration / training. Please see the linked configs in the table for specific settings such as batch size, FSDP, activation checkpointing, optimizer, etc used to obtain the peak memory. The specific HW resources specified are meant as an example for possible hardware that can be used.
+Memory-efficiency is important to us. All of our recipes are tested on a variety of setups including commodity GPUs with 24GB of VRAM as well as beefier options found in data centers.
```

Comment:
Super nit but I don't think memory efficiency should be hyphenated here

Reply:
good call!